src/models/DefaultFilters.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const ElectronConfig = require('electron-config')
+const configName = process.env.NODE_ENV === 'test' ? 'config-test' : 'config'
+const storage = new ElectronConfig({ name: configName })
+const Filter = require('./Filter')
+const LastFilter = require('./LastFilter')
+const DEFAULT_FILTER_KEY = 'default-filters'
+
+const DEFAULT_FILTERS = {
+  'My Issues': 'author:login is:open sort:updated-desc',
```

Comment:
Could similarly add a 'My Pull Requests' filter for the user to keep track of when their PRs get new activity.